navbar.css

```diff
@@ -0,0 +1,40 @@
+#navbar{
+    height: 50px;
+    display: grid;
+    grid-template-columns: 100px 100px;
+    grid-gap: 10px;
+    position: fixed;
+    top:0px;
+    left: 0px;
+    right: 0px;
+    z-index: 100;
+    background-color: rgb(138, 46, 46);
+    
+}
+#github,
+#menu{
+  font-family: Arial, Helvetica, sans-serif;
+  color: white;
+  font-weight: bold;
+  justify-content: center;
+  transition: 0.15s;
+  margin-left: 10px;
+}
+#githubText{
+  text-decoration: none;
+}
+#github :hover,
+#menu :hover{
+cursor: pointer;
+color: rgb(199, 199, 199);
+}
+a:link {
+  color: white;
+  text-decoration: none;
+}
```

Comment:
You missed a blank line between each one of these selectors. Kindly add the blank lines to make your code more readable.